api/tests/routes/backoffice/multiple_offers_test.py

```diff
@@ -334,3 +338,37 @@ def test_edit_product_gcu_compatibility(self, authenticated_client, validation_s
             else:
                 assert offer.lastValidationType == OfferValidationType.CGU_INCOMPATIBLE_PRODUCT
                 assert datetime.datetime.utcnow() - offer.lastValidationDate < datetime.timedelta(seconds=5)
+
+    @pytest.mark.parametrize(
+        "validation_status",
+        [
+            OfferValidationStatus.DRAFT,
+            OfferValidationStatus.PENDING,
+            OfferValidationStatus.APPROVED,
+        ],
+    )
+    def test_send_mail_when_edit_product_gcu_compatibility(self, authenticated_client, validation_status):
+
+        provider = providers_factories.APIProviderFactory()
+        product_1 = offers_factories.ThingProductFactory(
+            description="premier produit inapproprié",
+            extraData={"ean": "9781234567890"},
+            isGcuCompatible=not validation_status == OfferValidationStatus.REJECTED,
```

Comment:
Tu n'as jamais le cas `REJECTED`, inutile de compliquer le code pour rien
```suggestion
            isGcuCompatible=True,
```